administrator/components/com_joomlaupdate/src/Model/UpdateModel.php

```diff
@@ -483,7 +579,7 @@ protected function downloadPackage($url, $target)
         // Download the package
         try {
             $result = HttpFactory::getHttp([], ['curl', 'stream'])->get($url);
-        } catch (\RuntimeException $e) {
+        } catch (RuntimeException $e) {
```

Comment:
```suggestion
        } catch (\RuntimeException $e) {
```
Revert this change because in J4 the file is namespaced.